lib/mix/test/test_helper.exs

```diff
@@ -3,6 +3,32 @@ Mix.shell(Mix.Shell.Process)
 Application.put_env(:mix, :colors, [enabled: false])
 ExUnit.start [trace: "--trace" in System.argv]
 
+git_version =
+  case Mix.State.fetch(:git_version) do
```

Comment:
Let's make Mix.SCM.Git.git_version public. It is fine to use it in our tests since the whole module is `@moduledoc false` anyway.